apps/jobs/utils.py

```diff
@@ -0,0 +1,32 @@
+from rest_framework import status
+from rest_framework.response import Response
+from challenges.models import (
+    ChallengePhase,
+    Challenge,
+)
+from participants.models import (ParticipantTeam,)
+
+
+def get_challenge_object(challenge_pk):
+    try:
+        Challenge.objects.get(pk=challenge_pk)
+    except Challenge.DoesNotExist:
+        response_data = {'error': 'Challenge does not exist'}
+        return Response(response_data, status=status.HTTP_400_BAD_REQUEST)
+
+
+def get_challenge_phase_object(challenge_phase_pk):
```

Comment:
@RishabhJain2018 : Since with time these functions will grow, so we can create these functions dynamically(can be kept in base app) and then in utils of each app, call that function to get `get_*_object` function.

This way we prevent ourselves from complaints of code climate also 😄